source/atlas-cli-ephemeral-cluster.txt

```diff
@@ -0,0 +1,115 @@
+.. _atlas-cli-ephemeral-cluster:
+
+=====================================================
+Test Automations with Ephemeral Projects and Clusters
+=====================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This tutorial demonstrates how to quickly create and delete ephemeral 
+|service| projects and {+clusters+} using the {+atlas-cli+}. Ephemeral
+projects and {+clusters+} provide temporary testing environments that
+are isolated from your production projects and {+clusters+}. You can use
+ephemeral projects and {+clusters+} to test your automation scripts
+before using the scripts on production {+clusters+}.
+
+.. _atlas-cli-ephemeral-reqs:
+
+Prerequisites 
+-------------
+
+Before you begin, complete the following tasks: 
+
+- Create an |service| user account
+- :ref:`Create <atlas-organizations-create>` an 
+  :atlas:`Atlas organization </organizations-projects/>` and
+  note its organization ID. You can find the organization ID by running the :ref:`atlas-organizations-list` command.
+- :ref:`install-atlas-cli`.
+- :ref:`connect-atlas-cli`.
+
+Follow These Steps
+------------------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Create the ephemeral project, {+cluster+}, and database user.
+
+      a. Run the :ref:`atlas-users-describe` command to find your 
+         |service| user ID. Replace <YOUR-EMAIL> with the email address
```

Comment:
question: is this returned in a user ID field? Might be worth showing example output